packages/storage/src/indexed-db/config.ts

```diff
@@ -2,4 +2,4 @@
  * The version number for the IndexedDB schema. This version number is used to manage
  * database upgrades and ensure that the correct schema version is applied.
  */
-export const IDB_VERSION = 46;
+export const IDB_VERSION = 47;
```

Comment:
Necessary to bump storage version as users may have NFTs (auction/LP positions) that are misattributed to them as being the owner.